tests/conftest.py

```diff
@@ -134,12 +134,10 @@ def teardown():
         signpost.terminate()
         wait_for_signpost_not_alive(port)
 
-    _app.config.from_object("sheepdog.test_settings")
+    _app = app_init()
 
     request.addfinalizer(teardown)
 
-    app_init(_app)
-
     _app.logger.setLevel(os.environ.get("GDC_LOG_LEVEL", "WARNING"))
 
     return _app
```

Comment:
FYI this fixture is functionally scoped so it is both writing to a tmp file and starting and signpost process with any test that references this.

This problem is excaerbated because the pytest-flask fixture client should depend on app. Not part of this PR, but it is why the API tests were so slow 6 months ago.